src/main/java/core/basesyntax/service/TransactionsFileReaderService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Transaction;
+import java.util.List;
+
+public interface TransactionsFileReaderService {
```

Comment:
remove this interface and its implementation below, you already have an implementation where you can pass a path to file to read it